telegram/ext/_callbackqueryhandler.py

```diff
@@ -55,8 +55,8 @@ class CallbackQueryHandler(BaseHandler[Update, CCT]):
         When setting :paramref:`block` to :obj:`False`, you cannot rely on adding custom
         attributes to :class:`telegram.ext.CallbackContext`. See its docs for more info.
 
-    .. seealso:: `Types of Handlers <https://github.com/\
-        python-telegram-bot/python-telegram-bot/wiki/Types-of-Handlers>`_,
+    .. seealso:: :any:`Nested Conversation Bot Example <examples.nestedconversationbot>`,
+        :any:`Arbitrary Callback Data Bot Example <examples.arbitrarycallbackdatabot>`
```

Comment:
arbitrary callback_data is already linked below - one is enough IMO. Nested conversations also work with other handlers, so I'd vote to remove this link here

Reply:
removing these 2 lines with links to Nested Conversation Bot and Arbitrary Callback Data Bot examples